### PR TITLE
{Release} Upgrade to Azure CLI 2.66.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,112 @@
...
+* `az bicep publish`: Remove preview flag from `--with-source` parameter (#29915)
+* `az data-boundary show`: Add command to support getting the data boundary at a specified scope (#29961)
+* `az data-boundary show-tenant`: Add command to support getting the data boundary at a tenant level (#29961)
+* `az data-boundary create`: Add command to support creating tenant to data boundary (#29961)
+* `az bicep generate-params`: Fix the help message for `--include-params` parameter (#30034)
+* `az bicep\deployment\stack`: Conditionally enable invariant globalization for Bicep running inside the Azure CLI docker image (#29897)
+
+**Backup**
+
+* Add support for enabling Disk access settings for managed VM restores (#29508)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:31]:
```suggestion
* `az backup restore restore-disks`: Add support for enabling Disk access settings for managed VM restores (#29508)
```

File: src/azure-cli-core/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+2.66.0
+++++++
+* : `aaz`: Support managed identity (#29953)
```
Comment (necusjz) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* `aaz`: Support managed identity (#29953)
```

File: src/azure-cli-core/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,17 @@
...
+* Resolve CVE-2024-6119 (#29887)
+* Resolve CVE-2024-5535 (#29887)
+* Resolve CVE-2024-4741 (#29887)
+* Resolve CVE-2024-4603 (#29887)
+* Resolve CVE-2024-2511 (#29887)
+* `az extension add/update` and `az upgrade`: Pre-announcement default value change from `true` to `false` for `--allow-preview` (#30162)
```
Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:14]: This should be put under `extension` section.

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,112 @@
...
+* `az functionapp plan update`: Do not show warning message with `--set` in command (#30144)
+
+**ARM**
+
+* `az bicep publish`: Remove preview flag from `--with-source` parameter (#29915)
+* `az data-boundary show`: Add command to support getting the data boundary at a specified scope (#29961)
+* `az data-boundary show-tenant`: Add command to support getting the data boundary at a tenant level (#29961)
+* `az data-boundary create`: Add command to support creating tenant to data boundary (#29961)
+* `az bicep generate-params`: Fix the help message for `--include-params` parameter (#30034)
+* `az bicep\deployment\stack`: Conditionally enable invariant globalization for Bicep running inside the Azure CLI docker image (#29897)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:27]:
```suggestion
* `az bicep/deployment/stack`: Conditionally enable invariant globalization for Bicep running inside the Azure CLI docker image (#29897)
```

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:31]:
```suggestion
* `az backup restore restore-disks`: Add support for enabling Disk access settings for managed VM restores (#29508)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,112 @@
...
+**SignalR**
+
+* `az signalr start/stop`: Add signalr command start or stop an existing SignalR service (#30000)
+* `az signalr replica start/stop/restart`: Add signalr command start, stop or start an existing SignalR service (#30058)
+* `az signalr network-rule ip-rule add/remove`: Add signalr command add or remove ip-rule of an SignalR service (#30058)
+* `az signalr update client-cert-enabled/disable-local-auth/region-endpoint-enabled`: Toggle clientCertificate, disableLocalAuth, enableRegionEndpoint (#30116)
+
+**SingalR**
+
+* `az signalr replica update region-endpoint-enabled/unit-count`: Toggle enableRegionEndpoint and change unitCount (#30116)
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:101]:
```suggestion
* `az signalr replica update region-endpoint-enabled/unit-count`: Toggle enableRegionEndpoint and change unitCount (#30116)
```

File: src/azure-cli-core/HISTORY.rst (hunk 3 above)
Comment (AllyW) [inline src/azure-cli-core/HISTORY.rst:14]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,112 @@
...
+
+**RDBMS**
+
+* `az postgres flexible-server replica create`: Add support for `--tags` parameter (#29920)
+* `az postgres flexible-server replica create`: Allow read replica create from storage auto-grow enabled primary server flexible server (#30186)
+* `az postgres flexible-server backup create/delete`: New commands to support creating and deleting backups on PostgreSql Flex Server (#30197)
+
+**Service Bus**
+
+* `az servicebus namespace replica`: New command group to manage servicebus namespace replicas (#29987)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:85]:
```suggestion
* `az servicebus namespace replica`: Add new command group to manage servicebus namespace replicas (#29987)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,112 @@
...
+
+* `az vmss create/update`: Add new parameter `--skuprofile-vmsizes` to specify a list of VM sizes to use with VMSS Instance Mix (#29906)
+* `az vmss create/update`: Add new parameter `--skuprofile-allocation-strategy` to update the allocation strategy for VMSS Instance Mix (#29906)
+
+**Container app**
+
+* Fix #30053: `az containerapp update`: Fix polling (#30078)
+* `az containerapp env create`: Hide `--dapr-instrumentation-key` parameter (#30117)
+* `az containerapp env update`: Support to update `--dapr-connection-string` (#30117)
+
```
Comment (AllyW) [inline src/azure-cli/HISTORY.rst:43]:
```suggestion

** Extension**
* `az extension add/update`: Pre-announcement default value change from `true` to `false` for `--allow-preview` (#30162)

```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,112 @@
...
+**SQL**
+
+* `az sql geo-backup`: Remove preview tag (#30122)
+* `az sql failover-group create/update`: Add parameter `--secondary-type` to support geo/standby replica on partner server (#30060)
+
+**Storage**
+
+* `az storage blob copy start/start-batch`: Fix `--auth-mode login` (#29964)
+
```
Comment (AllyW) [inline src/azure-cli/HISTORY.rst:108]:
```suggestion

** Upgrade**
* `az upgrade`: Pre-announcement default value change from `true` to `false` for `--allow-preview` (#30162)

```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,115 @@
...
+** Upgrade**
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:112]:
```suggestion
**Upgrade**
```

File: src/azure-cli/HISTORY.rst (hunk 5 above)
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst]:
```suggestion
* `az signalr update client-cert-enabled/disable-local-auth/region-endpoint-enabled`: Toggle clientCertificate, disableLocalAuth, enableRegionEndpoint (#30116)
* `az signalr replica update region-endpoint-enabled/unit-count`: Toggle enableRegionEndpoint and change unitCount (#30116)
```

**Review by necusjz — APPROVED** [review]

**Review by bebound — DISMISSED** [review]

**Review by evelyn-ys — DISMISSED** [review]

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,115 @@
...
+* `az vmss create/update`: Add new parameter `--skuprofile-vmsizes` to specify a list of VM sizes to use with VMSS Instance Mix (#29906)
+* `az vmss create/update`: Add new parameter `--skuprofile-allocation-strategy` to update the allocation strategy for VMSS Instance Mix (#29906)
+
+**Container app**
+
+* Fix #30053: `az containerapp update`: Fix polling (#30078)
+* `az containerapp env create`: Hide `--dapr-instrumentation-key` parameter (#30117)
+* `az containerapp env update`: Support to update `--dapr-connection-string` (#30117)
+
+** Extension**
```
Comment (AllyW) [inline src/azure-cli/HISTORY.rst:44]:
```suggestion
**Extension**
```

**Review by zhoxing-ms — DISMISSED** [review]

**Review by bebound — APPROVED** [review]

**Review by yanzhudd — DISMISSED** [review]

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,115 @@
...
+* `az vmss create/update`: Add new parameter `--skuprofile-vmsizes` to specify a list of VM sizes to use with VMSS Instance Mix (#29906)
+* `az vmss create/update`: Add new parameter `--skuprofile-allocation-strategy` to update the allocation strategy for VMSS Instance Mix (#29906)
+
+**Container app**
+
+* Fix #30053: `az containerapp update`: Fix polling (#30078)
+* `az containerapp env create`: Hide `--dapr-instrumentation-key` parameter (#30117)
+* `az containerapp env update`: Support to update `--dapr-connection-string` (#30117)
+
+**Extension**
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:44]:
```suggestion
**Extension**

```

File: src/azure-cli/HISTORY.rst [hunk 12]
```diff
@@ -3,6 +3,115 @@
...
+**SQL**
+
+* `az sql geo-backup`: Remove preview tag (#30122)
+* `az sql failover-group create/update`: Add parameter `--secondary-type` to support geo/standby replica on partner server (#30060)
+
+**Storage**
+
+* `az storage blob copy start/start-batch`: Fix `--auth-mode login` (#29964)
+
+**Upgrade**
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:112]:
```suggestion
**Upgrade**

```

**Review by bebound — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**